birdhouse/env.local.example

```diff
@@ -263,7 +263,8 @@ export GEOSERVER_ADMIN_PASSWORD="${__DEFAULT__GEOSERVER_ADMIN_PASSWORD}"
 #export ALLOW_UNSECURE_HTTP=""
 
 # Jupyter single-user server images
-#export DOCKER_NOTEBOOK_IMAGES="pavics/workflow-tests:210216 \
+#export DOCKER_NOTEBOOK_IMAGES="marbleclimate/marble-jupyter-images \
```

Comment:
Where is this hosted? I cannot find them on DockerHub (default for `docker pull`). Is it on a private/other registry?
If a user enables this as is without appropriate `docker login` to the appropriate registry, that will make the deployment fail due to unresolved reference, unless the Jupyter config is overridden here to allow missing dockers : 
https://github.com/bird-house/birdhouse-deploy/blob/eaa09c2819e5e5dc5bd36dae6827562f3beef1c5/birdhouse/components/jupyterhub/jupyterhub_config.py.template#L181

Also, just to make sure, is the trailing `s` of the image a typo? Is it a single docker image using this name explicitly? Images should be listed individually here.